control-plane-operator/controllers/hostedcontrolplane/cno/clusternetworkoperator_test.go

```diff
@@ -4,7 +4,13 @@ import (
 	"strconv"
 	"testing"
 
+	. "github.com/onsi/gomega"
+	routev1 "github.com/openshift/api/route/v1"
+	hyperv1 "github.com/openshift/hypershift/api/hypershift/v1beta1"
```

Comment:
this path is for 4.15 and later.  4.14 is still `github.com/openshift/hypershift/api/v1beta1`.